src/web/main.cpp

```diff
@@ -103,6 +108,12 @@ int main(int argc, char **argv)
         printf("interfaceName not specified, using default %s\n", interfaceName);
     }
 
+    if (httpListenAddr == NULL)
+    {
+        httpListenAddr = kDefaultListenAddr;
+        printf("litenAddress not specified, using default %s\n", httpListenAddr);
```

Comment:
use `stderr` for error message. typo: `litenAddress` -> `Bind address`.

Reply:
It's just a info, so I think stdout suffices.